public/files/ads/ads.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-var
```

Comment:
This doesn't need to be here, it's not currently checked by ESLint in my configuration, and this file doesn't get minified so adding a comment results in extra bytes sent to the user.